qutip/control/dump.py

```diff
@@ -1,37 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
We should keep these encoding lines.

Reply:
Noted in the main response, but these encoding lines are an artifact of Python 2, and not actually required any more.  Regardless, I've left them in because they aren't in scope of this PR.